omod/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>providermanagement</artifactId>
-		<version>2.14.0-SNAPSHOT</version>
```

Comment:
What is this version change for?

Reply:
https://github.com/openmrs/openmrs-module-providermanagement/pull/45#discussion_r1379418520

i was reacting to this comment, and was restoring the prevoius version before my commits 
intially i had som build failures thats why i made  switch